dm/tests/checkpoint_transaction/run.sh

```diff
@@ -28,6 +58,29 @@ function run() {
 	# check diff
 	check_sync_diff $WORK_DIR $cur/conf/diff_config.toml
 
+	# test ungraceful stop, worker will not wait transaction finish
+	run_sql_file $cur/data/db1.increment1.sql $MYSQL_HOST1 $MYSQL_PORT1 $MYSQL_PASSWORD1
+	sleep 2
+	# kill dm-master 1 to make worker lost keepalive while a transaction is not finished
+	echo "kill dm-master1"
+	kill_dm_master
+	check_master_port_offline 1
+	sleep 1 # wiat woker lost ha keepalive-ttl is 1 second
```

Comment:
```suggestion
	sleep 1 # wait woker lost ha keepalive-ttl is 1 second
```

please check typo in this file